README.md

```diff
@@ -22,7 +22,13 @@ accept.
 To create a CFP, it is recommended to use the `CFP-003-template.md`
 file as an outline. The structure of this template is meant to provide a starting
 point for people. Feel free to edit and modify your outline to best fit your
-needs when creating a proposal. When you are ready to submit your CFP please:
+needs when creating a proposal.
+
+Many design docs also begin their life as a Google doc or other shareable
+file for easy commenting and editing when still in the early stages of discussion.
+Once your proposal is done, submit it as a PR to the design-cfps folder.
```

Comment:
```suggestion
Once your proposal is ready, submit it as a PR to the design-cfps folder for the project it is for.
```

Reply:
hmm, there aren't any designated folders, they are just all under the `cilium` folder

Reply:
I was thinking that each project would have its own folder. It just hasn't happened yet since there haven't been CfPs for hubble or tetragon yet

Reply:
ah gotcha - in that case, until the folders are made, i think we should keep it as is, make the change to the wording once they are present?

Reply:
Maybe we can just add the folders in this PR too?